src/handler/src/GitHub.types.ts

```diff
@@ -0,0 +1,90 @@
+export interface GithubAuthor {
```

Comment:
I don't think the file should have `.types.` in its name, rather make a folder called `interfaces` or something.

Also: `GitHubAuthor`